bundle-workflow/src/manifests/test_manifest.py

```diff
@@ -21,6 +21,8 @@ class TestManifest(Manifest):
                 - with-security
                 - without-security
                 - with-less-security
+              additional-cluster-configs:
```

Comment:
Maybe introduce a config: key with cluster: and tests: children?

Reply:
I'm ok with issues being opened against this.

Reply:
... but while I am OK with moving forward adding this config _is_ the entire point of this PR. So we shouldn't be acquiring as much technical debt here.